.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+/       @heartsucker @redshiftzero @creviera
```

Comment:
I'm not sure this is what we want - since these [codeowner files](https://help.github.com/en/articles/about-code-owners) use [gitignore syntax](https://git-scm.com/docs/gitignore#_pattern_format), shouldn't this `/` be `*` for a global review request?

Reply:
Fixed.